beta/iam/public-classes-enums.md

```diff
@@ -0,0 +1,219 @@
+# Class - Message
+
+The `Message` class contains the definition of an in-app message and controls its tracking via Experience Edge events.
+
+`Message` objects are only created by the AEPMessaging extension, and passed as the `message` parameter in `MessagingDelegate` protocol methods.
+
+## Public variables
+
+### id
+
+Identifier of the `Message`. This value matches the Message Execution ID assigned by AJO Campaign.
```

Comment:
Since this is the first mention of Adobe Journey Optimizer within this document, it should be referred to as its full name, as such "... Adobe Journey Optimizer (AJO) Campaign."